spec/e2e/pbivizStartSpec.js

```diff
@@ -26,13 +26,12 @@
 
 "use strict";
 
-const fs = require('fs-extra');
-const path = require('path');
-const async = require('async');
-const FileSystem = require('../helpers/FileSystem.js');
-const writeMetadata = require("./utils").writeMetadata;
-const download = require("../../lib/utils").download;
-const createFolder = require("../../lib/utils").createFolder;
+import fs from 'fs-extra';
+import path from 'path';
+import async from 'async';
+import FileSystem from '../helpers/FileSystem.js';
+import { writeMetadata } from "./utils.js";
+import { download, createFolder } from "../../lib/utils.js";
```

Comment:
what's the difference between ./utils.js and ../../lib/utils.js. 
Is that the same lib? it looks confusing.

Reply:
It's two different files. The first one is for only test utils, so I decided to rename it to testUtils.